infra/build/functions/deploy.sh

```diff
@@ -124,4 +134,11 @@ deploy_cloud_function request-coverage-build \
 					  $COVERAGE_BUILD_JOB_TOPIC \
 					  $PROJECT_ID
 
+
+deploy_cloud_function request-introspector-build \
+						introspector_build \
```

Comment:
nit: fix indent.

Reply:
Done.

Reply:
Think you forgot to push. I edited this file.